app/controllers/dashboard/associated_users_controller.rb

```diff
@@ -71,6 +71,10 @@ def create
     creator = Dashboard::AssociatedUserCreator.new(params[:project_role])
 
     if creator.successful?
+      if @current_user_created = params[:project_role][:identity_id].to_i == @user.id
+        @permission_to_edit = creator.protocol_role.can_edit?
+      end
+
       flash.now[:success] = 'Authorized User Added!'
```

Comment:
If the current user is created, we may need to re-render service requests tables to show Modify Request buttons and the Edit buttons